unlock-app/src/components/interface/Error.js

```diff
@@ -31,7 +31,7 @@ const mapStateToProps = ({ errors }) => ({
 
 const mapDispatchToProps = dispatch => ({
   close: () => {
-    dispatch(setError())
+    dispatch(resetError())
```

Comment:
Wouldn't it make more sense here to actually reset the error being displayed? (Since the other ones are not displayed I think we should not delete them until we know the user has seen them)